design-system/uilib/src/components/dpdhl-toast/dpdhl-toast.svelte

```diff
@@ -0,0 +1,127 @@
+<svelte:options tag="dpdhl-toast" />
+
+<script lang="ts">
+	/**
+	 * A Toast Component
+	 * @component
+	 */
+
+	/**
+	 * Type of the toast: Error, Notification
+	 */
+	import { Type } from "./type";
+
+	export let open = true;
+	$: _open = open;
+
+	export let message = "";
+	$: _message = message;
+
+	export let type: string = Type.error;
+	export let fullWidth: boolean = true;
+
+	export let title;
+	$: _title = title || "No title provided";
```

Comment:
should this just be an empty string?
could there be a use case where as a user you don't want to provide a title?